lighthouse-core/gather/driver.js

```diff
@@ -69,17 +69,17 @@ class Driver {
      */
     this._monitoredUrl = null;
 
-    let targetProxyMessageId = 0;
     this.on('Target.attachedToTarget', event => {
-      targetProxyMessageId++;
       // We're only interested in network requests from iframes for now as those are "part of the page".
       if (event.targetInfo.type !== 'iframe') return;
 
       // We want to receive information about network requests from iframes, so enable the Network domain.
       // Network events from subtargets will be stringified and sent back on `Target.receivedMessageFromTarget`.
```

Comment:
We still get back these messages over the target domain?

In https://github.com/GoogleChrome/puppeteer/pull/3827/files#diff-0c1fe9dd78cc7f29823c09b251c8aae0R109 I saw the sessionId being received and handled in the `handleRawMessage` method (basically).

Reply:
whoops no, I need to update the comment 👍